models/user.php

```diff
@@ -0,0 +1,81 @@
+<?php 
+require_once "base.php";
+require_once "../../class/users/class_php.php";
+/*
+ * This class inherits from the base class and contains the calls to the posts procedures
+ * 
+ * The model do not have to be called in other file than the posts_controller
+ * Info: The model file name must be in singular and be in snake case, the class name must be
+ *       in camel case with the first letter in uppercase and inherits the base class
+ * 
+ */
+class User extends Base {
+    /**
+     * The constructor is used to connect to the database
+     * 
+     * @param void
+     * @throws Exception if it fails to connect to the database
+     * @return void
+     */
+public function __construct() {
+        try {
+            $this->conn = $this->db_connection();
+            $this->classPHP = new ClassPHP();
+        } catch (Exception $e) {
+            throw new Exception("Failed to connect to the database: " . $e->getMessage());
+        }
+    }
+
+    /** 
+     * Get all the posts in the database and each image by post
+     * 
+     * @param void
+     * @throws PDOException if it fails to execute the query
+     * @throws Exception if it fails to get all save user
+     * @return array
+     */
+
+
+    public function save($data) {
+        $classPHP = new ClassPHP();
```

Comment:
do not create a instance of a class here, i already leave a comment in class_php saying that if you are using that class just here, then move the methods to this class as private or public(depends of the way you plan to using it)